lib/teamsnap/api.rb

```diff
@@ -73,6 +73,8 @@ def self.via(klass, method)
 
     def self.parse_error(resp)
       return "Object Not Found (404)" if resp.status == 404
+      return "Forbidden (403)" if resp.status == 403 && resp.body == ""
```

Comment:
We have some cases in apiv3 where a 403 does return it's own body so in those cases I want to return what apiv3 generated. I only need to check if there isn't a body.
I wonder if 404 should be checking for an empty body. There are some instances of apiv3 returning a 404 and having an error message. @glanotte